dynamorm/table.py

```diff
@@ -255,12 +255,13 @@ def scan(self, scan_kwargs=None, **kwargs):
             assert len(parts) == 0, "Left over parts after parsing query attr"
 
             op = getattr(attr, op)
+            op_fn = op(value) if value is not None else op()
```

Comment:
FWIW only the `exists`, `not_exists` and `size` conditions don't take a value.

http://boto3.readthedocs.io/en/latest/_modules/boto3/dynamodb/conditions.html

[The `size` condition says](http://boto3.readthedocs.io/en/latest/_modules/boto3/dynamodb/conditions.html#Attr.size):

> Note another AttributeBase method must be called on the returned
> size condition to be a valid DynamoDB condition.

So we will probably need to support the size condition in some special way.  I'll open another issue for it to track it and we can tackle it once a use-case for the size condition comes up.